config/locales/server.en.yml

```diff
@@ -1805,7 +1805,7 @@ en:
     logo: "The logo image at the top left of your site. Use a wide rectangular image with a height of 120 and an aspect ratio greater than 3:1. If left blank, the site title text will be shown."
     logo_small: "The small logo image at the top left of your site, seen when scrolling down. Use a square 120 × 120 image. If left blank, a home glyph will be shown."
     digest_logo: "The alternate logo image used at the top of your site's email summary. Use a wide rectangle image. Don't use an SVG image. If left blank, the image from the `logo` setting will be used."
-    mobile_logo: "The logo used on mobile version of your site. Use a wide rectangular image with a height of 120 and an aspect ratio greater than 3:1. If left blank, the image from the `logo` setting will be used."
+    mobile_logo: "The logo used on mobile version of your site. You can use a wide rectangular image with a height of 120 and an aspect ratio greater than 3:1, but we recommend a square icon to avoid crowding the header. If left blank, the image from the `logo` setting will be used."
```

Comment:
```suggestion
    mobile_logo: "The logo used on mobile version of your site. We recommend a square icon to avoid crowding the header, but you can use a wide rectangular image with a height of 120 and an aspect ratio greater than 3:1. If left blank, the image from the `logo` setting will be used."
```

Reply:
I think we should start with our recommendation of a square logo as the first part of the sentence.